Sources/SwiftDocC/Semantics/Metadata/AlternateRepresentation.swift

```diff
@@ -0,0 +1,85 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2024 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See https://swift.org/LICENSE.txt for license information
+ See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+import Foundation
+import Markdown
+
+
+/// A directive that configures the alternate language representations of a symbol.
+///
+/// Defines an alternate language representation for the current symbol, such that both symbols are considered to be alternate representations of the same symbol,
+/// and are equivalent, even if they are different symbols according to the compiler.
+/// These two symbols should not have any source languages in common.
+/// ```md
+/// @Metadata {
+///     @AlternateRepresentation(``MyApp/MyClass/property``)
+/// }
+/// ```
+///
+/// External links must be wrapped in quotes:
```

Comment:
> I think this statement is false.

This is part of the behaviour of directive argument parsing, which is why I added a mention. Unwrapped strings with a `:` within a directive are parsed as an argument name, such that `@AlternateRepresentation(doc://com.example/documentation/MyClass/property)` is parsed as an argument with name `doc` and value `//com.example/documentation/MyClass/property`, unless you wrap the whole thing in quotes. And since the directive is not expecting an argument `doc`, then the parsing fails with a `org.swift.docc.UnknownArgument` error.

Reply:
We call those "external documentation sources" to distinguish them from external DocC content. External documentation sources are used for bridging to documentation systems other than DocC.